packages/landingpage/components/getting-started/layout.tsx

```diff
@@ -13,11 +14,21 @@ interface Props {
   framework: Framework;
 }
 
+const SANDBOX_MAP: { [key in Framework]?: string } = {
+  [Framework.REACT]: '/codesandbox-react.png',
+  [Framework.ANGULAR]: '/codesandbox-angular.png',
+  [Framework.VUE]: '/codesandbox-vue.png',
+};
+
 const Layout = ({ children, framework, sandboxUrl }: Props) => {
   const { push } = useRouter();
   const frameworkName = NameByFramework[framework];
   const { t, locale } = useTranslation();
 
+  function handleImageClick() {
+    window.open(sandboxUrl, '_blank');
+  }
+
```

Comment:
Would it be more accessible to wrap the image in an anchor tag so that screen readers and such recognize it as link, i.e. `<a target="_blank" rel="noopener noreferrer"></a>`